.github/workflows/frontend.yaml

```diff
@@ -24,6 +24,22 @@ jobs:
       - name: Install dependencies
         run: yarn --frozen-lockfile
 
+      - name: Download graph-federation tar.gz
+        run: wget https://github.com/DiamondLightSource/graph-federation/archive/refs/tags/1.0.0.tar.gz -O graph-federation.tar.gz
+
+      - name: Extract graph-federation archive
+        run: tar -xzf graph-federation.tar.gz
+
+      - name: Copy workflows.graphql to relay-workflows-lib
+        run: cp graph-federation-1.0.0/schema/workflows.graphql relay-workflows-lib/
+
+      - name: Remove directives from workflows.graphql
+        run: sed -i -e :a -e '$d;N;2,8ba' -e 'P;D' relay-workflows-lib/workflows.graphql
```

Comment:
Why do we need to remove the directives?

Reply:
relay compiler doesn't like those apollo directives, removing those doesn't hurt the schema